apps/docs/pages/guides/integrations/estuary.mdx

```diff
@@ -31,34 +31,32 @@ Before you begin, you'll need:
 
 You'll start by creating a **capture**, a task in Flow that connects to your data source system: in this case, Firestore. This process will create one or more data **collections**, backed by a real-time data lake.
 
-1. Go to the Flow web application at [dashboard.estuary.dev](http://dashboard.estuary.dev) and sign in using the credentials provided by your Estuary account manager.
+1. Go to the [**Captures** tab](https://dashboard.estuary.dev/captures) of the Flow web app and choose **New Capture**. 
 
-2. Click the **Captures** tab and choose **New Capture**.
-
-3. Locate and select the **Google Firestore** card.
+2. Locate and select the **Google Firestore** card.
 
    A form appears with the properties required for a Firestore capture.
 
-4. Set a name for your capture.
+3. Set a name for your capture.
 
    Click inside the **Name** field to generate a drop-down menu of available **prefixes** and select one (likely, this will be the name of your organization). Append a unique capture name after the `/` to create the full name, for example, `acmeCo/myFirestoreCapture`.
 
-5. Fill out the required properties for Firestore.
+4. Fill out the required properties for Firestore.
 
    - **Database**: Flow can autodetect the database name, but you may optionally specify it here. This is helpful if the service account used has access to multiple Firebase projects. Your database name usually follows the format `projects/$PROJECTID/databases/(default)`.
    - **Credentials**: The JSON service account key created per the prerequisites.
```

Comment:
Workaround for the broken numbering when an unordered list is nested within an ordered list:
```suggestion
   **Database**: Flow can autodetect the database name, but you may optionally specify it here. This is helpful if the service account used has access to multiple Firebase projects. Your database name usually follows the format `projects/$PROJECTID/databases/(default)`.
   
   **Credentials**: The JSON service account key created per the prerequisites.
```